luatest/assertions.lua

```diff
@@ -127,14 +127,14 @@ end
 -- @string message
 function M.skip_if(condition, message)
     if condition and condition ~= nil then
-        utils.luatest_error('skip', message, 2)
+        utils.luatest_error('skip', message or 'No reason', 2)
     end
 end
 
 function M.run_only_if(condition, message)
     -- continue a running test if condition is met, else skip it
     if not (condition and condition ~= nil) then
-        utils.luatest_error('skip', prettystr(message), 2)
+        utils.luatest_error('skip', message or 'No reason', 2)
```

Comment:
Was: `prettystr(message)`
Become: `message or 'No reason'` — should it be `prettystr(message) or 'No reason'`?

Reply:
IMHO, no `prettystr` should be here. If you look into the assertions.lua file, you will see that `prettystr` is used in asserting functions to make pretty error messages. The `skip` and `skip_if` functions don't use it, so the `run_only_if` function shouldn't as well.

Reply:
or like `prettystr(message or 'No reason')`

Reply:
If we use the variant with `prettystr`, we will get message wrapped by `"` which looks ugly.

```
$ ./.rocks/bin/luatest -v test/unit/start_key_test.lua
Started on Tue Jan 25 11:50:14 2022
    start_key.engine:"memtx".index_type:"HASH".test_passing ... (0.002s) skip
...ankov/Workspace/expirationd/test/unit/start_key_test.lua:27: "(no reason specified)"
```

IMHO, we don't need `prettystr` at all.